### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by olitrussle**
Various copy

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ In his spare time Pete can be found in the gym, running, playing golf or watchin
     <li>{% include icon-linkedin.html username="jonny-wildey-1b928481" %}</li>
 </ul>
 
-The other Jonny has been working on software since 2013 when he realised a significant amount of his day to day work at the time could be rendered obsolete if he did so. He has been attempting to make his job obsolete ever since. Jonny can often be found evangelising [lodash](https://lodash.com/) and finding synonyms for words like 'rule'*. He is also in charge of the Ocado orders for some reason.
+The other Jonny has worked on software since 2013, when he realised that by doing so he could render a significant amount of his day-to-day work obsolete. He's been attempting to make his job obsolete ever since. Jonny can often be found evangelising [lodash](https://lodash.com/) and finding synonyms for words like 'rule'*. He's also in charge of our Ocado orders for some reason.
```
Comment (jonnyarnold) [inline about.md]: Maybe change one of the uses of "obsolete" here?

Comment (olitrussle) [inline about.md]: "He's been working hard to fulfil this prophecy ever since." ?

Comment (jonnyarnold) [inline about.md]: Sounds good! @jonnywildey?

Comment (jonnywildey) [inline about.md]: Whatever you guys like 🌈

**Pull request merged**